generators/app/index.js

```diff
@@ -248,7 +255,7 @@ module.exports = class extends Generator {
         }
 
         if (oConfig.codeassist) {
-            packge.devDependencies["@sapui5/ts-types"] = "^1.96.0"; //keep this line in sync with ui5.yaml version
+            packge.devDependencies["@sapui5/ts-types-esm"] = "latest"; //keep this line in sync with ui5.yaml version
```

Comment:
shouldn't this be included based on the scenario? 

if enableTypescript it should always add the devDependencies to the esm types

but if codeassist and !enableTypescript it should add the classic ts-types.

Reply:
here we should also ensure that either or is included

Reply:
>but if codeassist and !enableTypescript it should add the classic ts-types.

I misunderstood the guidance at https://github.com/SAP/ui5-typescript#how-to-obtain-the-ui5-typescript-signatures. I read "using this flavor should be avoided" and thought that it shall not be used anymore

Reply:
Reverted with https://github.com/ui5-community/generator-ui5-project/pull/58/commits/0fc2353416b540d0be8eed26993cc843f726cd8b